### DISCUSSION TITLE
Feature suggeston

### DISCUSSION THREAD

**Discussion by mstick99** [lart2150/kmttg#19] [Ideas]
Add the ability to compare deleted files on Tivo to kmttg copied files and have option to delete or keep them.

**Comment by lart2150** [comment]
Can you expand on this some more? Do you want it to show what deleted shows you have downloaded kind of like it does on NPL?

**Comment by mstick99** [comment]
Hi
Here's what I was thinking.

Day 1: Tivo records 10 shows
Day 2: KMTTG is run coping the 10 shows to a directory
Day 3: Tivo records 7 shows (of which 1 is deleted), and 5 Day 1 shows are
deleted
Day 4: KMTTG is run deleting the 5 Day 1 shows in the KMTTG directory that
were deleted on Tivo, and copies the 6 (=7 recorded minus 1 deleted) Day 2
shows to the directory

The deletion of the shows should, I'm thinking, probably come first to free
up space and would be a quick process. All files that have the same show
name should also be deleted. This would keep the Tivo and KMTTG directory
somewhat in sync with each other. This would eliminate having to "clean
out" the KMTTG directory.

**Comment by mstick99** [comment]
Decided not to implement?

**Comment by mstick99** [comment]
Maybe this could be implemented by comparing the NPL list generated from the Tivo to the remote directory's files. Any file not found in the NPL list would then be deleted. Include a message/warning to move any remote file that the user wants to keep.
